ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -101,8 +237,129 @@ class ClientQueryViewController: UITableViewController {
 			self.navigationController?.pushViewController(ClientQueryViewController.init(core: self.core!, query: OCQuery.init(forPath: rowItem.path)), animated: true)
 		}
 	}
+
+	// MARK: - Message
+	var messageView : UIView?
```

Comment:
Yeah. I plan on doing that. And also use it for the "Welcome"-message in the Server List. Since the latter requires a bit of work on getting the constraints exactly right, I saved it for the next pull request.

Reply:
=> will do in a future branch.